clients/venice-client/src/test/java/com/linkedin/venice/fastclient/meta/VeniceClientBasedMetadataTest.java

```diff
@@ -0,0 +1,5 @@
+package com.linkedin.venice.fastclient.meta;
+
+public class VeniceClientBasedMetadataTest {
```

Comment:
Empty test file?

Reply:
The commit didn't include the latest additions, let me fix that.